vhdbuilder/release-notes/AKSWindows/2019-containerd/17763.1935.210513.txt

```diff
@@ -0,0 +1,136 @@
+﻿Build Number: 20210513.2_master_42463841

```

Comment:
why the release note is still 2019?

Reply:
that's because of the base os version?

Reply:
Yes. The base image is still 2019. The differences are
1. containerd is installed instead of docker
2. cached images are for containerd